docs/getting_started/aws.rst

```diff
@@ -159,44 +177,59 @@ AMI id that is available in the ``region`` your specified.
 This value will be dependent on the ``source_ami`` that you use. Common values
 are ``centos`` or ``ec2-user``.
 
-``datacenter`` is a name to identify your datacenter, this is important if you have more than one datacenter. 
+``datacenter`` is a name to identify your datacenter, this is important if you have more than one datacenter.
 
-``short_name`` is appended to the name tag and dns (if used) of each of the nodes to help better identify them. 
-
-``control_count``, ``edge_count`` and ``worker_count`` are the number of EC2 instances that will get deployed for each node type. 
+``control_count``, ``edge_count`` and ``worker_count`` are the number of EC2 instances that will get deployed for each node type.
 
 ``control_type``, ``edge_type`` and ``worker_type`` are used to specify the `EC2 instance type <https://aws.amazon.com/ec2/instance-types/>`_
 for your control nodes and worker nodes and they must be compatible with the
 ``source_ami`` you have specified. The default EC2 instance type is an m3.medium.
 
+Security Setup
```

Comment:
I'm not actually sure that it blocks terraform... [in our current getting started guide](http://docs.mantl.io/en/latest/getting_started/index.html), it is listed after "provisioning cloud hosts", and for instance on Vagrant, it is run after the hosts are created. 

That's a good mindset for writing docs, we should all try to keep that in mind.